config/uv/uv-config-reshare.json

```diff
@@ -0,0 +1,15 @@
+{
+  "modules": {
+    "footerPanel": {
+      "options": {
+        "downloadEnabled": false,
+        "shareEnabled": false
+      },
+      "contentLeftPanel": {
+        "options": {
+          "pageModeEnabled": true
```

Comment:
I think the two page view is located in the header and not the left panel

Reply:
yes good call, im just realizing its not working because my work with more than one page finally loaded. 

the line you had sent from nnp had it in this left panel thing, but 
theres also another pageModeEnabled in pagingHeaderPanel, but its set to false. should it be true?

https://github.com/scientist-softserv/nnp/blob/de9cf2427dff3ab6afebd9e3549b75bb5bd546c2/config/uv/uv-config.json#L196